src/components/HomepageFeatures/index.js

```diff
@@ -1,46 +1,40 @@
-import React from 'react';
-import clsx from 'clsx';
-import styles from './styles.module.css';
+import React from "react";
```

Comment:
Do we have a role for this change? Can we void this change?

Reply:
They appear as remove because of formatting they are re-added bellow, check
https://github.com/xprover/xprover-website/pull/1/files#diff-495aacebfb4951861ae805ac52ddca21c4f3a8fe9e807b5bad07eaba844cb77a